src/odemis/acq/align/test/align_test.py

```diff
@@ -0,0 +1,203 @@
+
+        self.data = tiff.read_data(os.path.join(TEST_IMAGE_PATH, "brightlight-off-slit-spccd-simple.ome.tiff"))
+        self.img_light_off = img.ensure2DImage(self.data[0])
+
+    def test_slit_off_on(self):
+        """
+        Test from dark slit image to bright slit image (gray)
+        """
+        bl = self.light
+        ccd = mock.FakeCCD(0, self.img_light_off)
+        f = turnLightAndCheck(bl, ccd)
+        time.sleep(1)
+        self.assertFalse(f.done())
+
+        # simulate light turning on
+        self.data = tiff.read_data(os.path.join(TEST_IMAGE_PATH, "brightlight-on-slit-spccd-simple.ome.tiff"))
+        img_light_on = img.ensure2DImage(self.data[0])
+
+        ccd.fake_img = img_light_on
+        time.sleep(1)
+        self.assertTrue(f.done())
+        f.result()
+
+ #    @skip("skip")
+    def test_slit_timeout(self):
+        """
+        Test Timeout
+        """
+        bl = self.light
+        ccd = mock.FakeCCD(0, self.img_light_off)
+        f = turnLightAndCheck(bl, ccd)
+        # no change in the image
+        time.sleep(1)
+        self.assertFalse(f.done())
+        # with self.assertRaises(TimeoutError):
+        #     f.result(timeout=5)
+        with self.assertRaises(futures.TimeoutError):
+            f.result(timeout=5)
+        f.cancel()
+        logging.warning("Light doesn't appear to turn on after 60s")
+
+
+    def test_slit_cancel(self):
+        """
+        Test cancelling does cancel (relatively quickly)
+        """
+        bl = self.light
+        ccd = mock.FakeCCD(0, self.img_light_off)
+        f = turnLightAndCheck(bl, ccd)
+        time.sleep(1)
+        f.cancel()
+        self.assertTrue(f.cancelled())
+        with self.assertRaises(CancelledError):
+            f.result(5)
+
+    def test_1pixel_image(self):
+        """
+        Test from dark image to bright image of 1 pixel
+        """
+        bl = self.light
+
+        #dark 1x1 image
+        img_dark = numpy.zeros((1,1), dtype= numpy.uint8)  # type: ndarray
+        da_dark = model.DataArray(img_dark)
+        ccd = mock.FakeCCD(0, da_dark)
+
+        f = turnLightAndCheck(bl, ccd)
+        time.sleep(3)
+        self.assertFalse(f.done())
+
+        # simulate light turning on
+        # bright 1x1 image
+        img_bright = numpy.ones((1,1), dtype=numpy.uint8)
+        br_bright = model.DataArray(img_bright)
+        ccd.fake_img = br_bright
+
+        time.sleep(3)
+        self.assertTrue(f.done())
+        f.result()
+
+    def test_light_already_on(self):
+        """
+        Test when chamber light is already on
+        """
+        bl = self.light
+
+        # bright image - light on
+        self.data = tiff.read_data(os.path.join(TEST_IMAGE_PATH, "brightlight-on-slit-spccd-simple.ome.tiff"))
+        self.bright_image = img.ensure2DImage(self.data[0])
+        ccd = mock.FakeCCD(0, self.bright_image)
+
+        f = turnLightAndCheck(bl, ccd)
+        time.sleep(3)
```

Comment:
I don't understand why this test case passes right now. Can you explain? Might be a sign of a bug...